keyboard/src/modules/Keyboard.rb

```diff
@@ -281,6 +282,21 @@ def Import(settings, syntax = :keyboard)
       when :language
         keyboard = GetKeyboardForLanguage(settings["language"], keyboard)
       end
+
+      # Checking if the keymap exists. Either it is the real keymap name
+      # or an alias.
+      if !Keyboards.code(keyboard)
```

Comment:
I would avoid this nested conditional. Something like 

```ruby
found_keyboard = Keyboards.code(keyboard) || Keyboards.alias(keyboard)

if !found_keyboard
  Report(...)
  return false
end
```

should work. What do you think?

Reply:
I do not see a real  benefit. I have these nested condition in order to assign the keyboard alias new and not only for displaying the error message.
At least I am out of ideas how it should be better ;-)

Reply:
Hmmm, even more, it could be an isolated method `find_keyboard`

```ruby
# @param keyboard [String] alias or keymap
def find_keyboard(keyboard)
  Keyboards.code(keyboard) || Keyboards.alias(keyboard)
end

...

if !find_keyboard
  ...
```